api/v2alpha1/pipeline_spec.proto

```diff
@@ -1,5 +1,6 @@
 syntax = "proto3";
 
+option go_package = "github.com/kubeflow/pipelines/api/v2alpha1/go";

```

Comment:
Can this be named `github.com/kubeflow/pipelines/api/v2alpha1/pipeline_spec` instead of having go as the package name?

Reply:
@neuromage the folder already has a subfolder called python, that's why I used go instead.

Are you preferring an easier to use package name? What do you think about `github.com/kubeflow/pipelines/api/v2alpha1/go/pipeline_spec`?

Reply:
Thanks Yuan. Good point, I see the issue now. We could create a directory called `pipeline_spec` under `github.com/kubeflow/pipelines/api/v2alpha1/` and place the generated Go proto files there. Then the client can import it like
`import "github.com/kubeflow/pipelines/api/v2alpha1/pipeline_spec"` 

It's not symmetric with Python but maybe that's ok, so that we don't get the language appearing in the package name. WDYT?

Reply:
SGTM, I don't hold a strong opinion on this